pkgs/applications/version-management/gitlab/gitlab-workhorse/default.nix

```diff
@@ -3,13 +3,13 @@
 stdenv.mkDerivation rec {
   name = "gitlab-workhorse-${version}";
 
-  version = "8.7.0";
+  version = "8.8.1";
 
   src = fetchFromGitLab {
     owner = "gitlab-org";
     repo = "gitlab-workhorse";
     rev = "v${version}";
-    sha256 = "1zlngc498hnzbxwdjn3ymr0xwrnfgnzzhn9lyf37yfbjl8x28n3z";
+    sha256 = "02a1z2nky31acd17q2n4pi4jkbkr6x6frfi6qlcsmfza0x0kzpc0";
```

Comment:
for darwin, it looks like there's a different release of this:
```
hash mismatch in fixed-output derivation '/nix/store/kmrla6v72mv9gjk1pm9nwi7mlnsl4qis-source':
  wanted: sha256:02a1z2nky31acd17q2n4pi4jkbkr6x6frfi6qlcsmfza0x0kzpc0
  got:    sha256:0dfzib24a23z92jw2h3pzbl1d7m56rbf36l55mblwhx7nhan9v2g
```

Reply:
Huh? @LnL7, any thoughts?

Reply:
Fetching from gitlab should be the same on both darwin and linux. Maybe the used git binary behaves differently?

Reply:
very odd, i know, but that's what ofborg is tripping on

Reply:
doing a nix-review on my macbook yields spectacular failures all over the place from trying to build ruby packages :(

Reply:
@jonringer did it previously work on macos?

Reply:
looks like that's a no.... gitlab only supports linux. Now I'm curious as to why ofborg tried to instantiate it...